spec/lib/process_settings/testing/monitor_spec.rb

```diff
@@ -6,30 +6,4 @@
 
 describe ProcessSettings::Testing::Monitor do
   it_should_behave_like "Monitor", '<override>', Logger.new('/dev/null'), []
```

Comment:
Maybe a general `#[]` test should be in here too? Just that we can call it and get back the setting we expect?

Reply:
Added